apps/liveviewjs.com/README.md

```diff
@@ -14,15 +14,17 @@ $ yarn
 $ yarn start
 ```
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+This command starts a local development server and opens up a browser window. Most changes are reflected live without
+having to restart the server.
```

Comment:
@blimmer there are some lines like this that look like there is a newline added after 120chars?  Won't that break the resulting pages?

Reply:
Nope, markdown requires a blank line to create a new paragraph, so this should look visually equivalent.

Reply:
👍